config/dev.webpack.config.js

```diff
@@ -7,9 +7,9 @@ const { config: webpackConfig, plugins } = config({
   debug: true,
   deployment: process.env.BETA ? 'beta/apps' : 'apps',
   useProxy: true,
-  useCloud: true, // Until console.redhat.com is working
-  appUrl: process.env.BETA ? '/beta/staging/starter' : '/staging/starter',
-  env: process.env.BETA ? 'ci-beta' : 'ci-stable',
+  localChrome: '/Users/hq/SoftwareDev/arivepr/insights-chrome/build/',
+  appUrl: process.env.BETA ? '/beta/openshift/sandbox' : '/openshift/sandbox',
+  env: process.env.BETA ? 'stage-beta' : 'stage-stable',
```

Comment:
I'd change this so app devs can switch between `stage` and other env
```suggestion
  env: `${process.env.ENVIRONMENT || 'stage'}-${process.env.BETA ? 'beta' : 'stable'},
```